### ISSUE TITLE
Synchronize WebAuthn keys across devices

### ISSUE THREAD

**Issue by carloscabanero** [blinksh/blink#1574]
Passkeys and in general WebAuthn keys are automatically synchronized through iCloud through all your devices. Blink does not synchronize the public information it needs to make use of that key on other devices. We need to sync this, and probably do a review of what else we are now able to do since we last refactored this code.

**Comment by michaelyork** [comment]
Yes please! Just subscribed to Blink+ today (as a Classic user) just for this ability before I realized it’s not fully implemented. Happy to support, but would love to see this work :)

**Comment by carloscabanero** [comment]
We should check #1712, we may want to make it an option to enable synchronization of even regular keys.

**Comment by carloscabanero** [comment]
Referencing https://github.com/blinksh/blink/discussions/1875.

At the moment we are blocking this as support for WebAuthn keys is stale in the broader community.

Please comment in the discussion.
